cli/Valet/Nginx.php

```diff
@@ -102,6 +103,20 @@ function installNginxDirectory()
     }
 
     /**
+     * Check nginx.conf for errors.
+     */
+    private function lint()
+    {
+        $this->cli->quietly(
+            'sudo nginx -c '.static::NGINX_CONF.' -t',
+            function() use (&$error) {
```

Comment:
What's this `use (&$error)` here for? Doesn't look like that variable exists or is used.